cli/src/pcluster/resources/login_node/user_data.sh

```diff
@@ -89,7 +89,9 @@ write_files:
           "scheduler": "${Scheduler}",
           "stack_name": "${AWS::StackName}",
           "stack_arn": "${AWS::StackId}",
-          "use_private_hostname": "${UsePrivateHostname}"
+          "use_private_hostname": "${UsePrivateHostname}",
+          "auto_scaling_group_name": "${AutoScalingGroupName}",
```

Comment:
why do we need these in dna.json? If they are only used in user data they do not need to go in dna.json